cwms_radar_api/src/main/java/cwms/radar/formatters/json/JsonV1.java

```diff
@@ -42,7 +46,8 @@
 				   Clob.class,
 				   Clobs.class,
 				   TimeSeriesGroup.class,
-				   RecentValue.class
+				   RecentValue.class,
+				   RatingTemplate.class, RatingTemplates.class, RatingSpec.class, RatingSpecs.class
```

Comment:
Same as the comment in #165 should be JSONv2.

Reply:
Yep.